sql/07-query-friends-gabriel.sql

```diff
@@ -0,0 +1,13 @@
+SELECT 	
+	name 
+FROM 	
+	student.student
```

Comment:
This file looks good, but often in industry people would alias their tables here.  E.g. "student.student s" would alias the table "s"